numpy/_core/_ufunc_config.pyi

```diff
@@ -15,14 +15,6 @@ class _ErrDict(TypedDict):
     under: _ErrKind
     invalid: _ErrKind
 
-@type_check_only
-class _ErrDictOptional(TypedDict, total=False):
-    all: None | _ErrKind
-    divide: None | _ErrKind
-    over: None | _ErrKind
-    under: None | _ErrKind
-    invalid: None | _ErrKind
-
```

Comment:
Curious as to why this is removed. Unused?

Reply:
Exactly. From the relevant commit:
> PYI049 Private TypedDict is never used